### DISCUSSION TITLE
wrong username and password on ALL-IN-ONE VM IMAGE

### DISCUSSION THREAD

**Discussion by HelloMrK** [nextcloud/all-in-one#1444] [Questions] [answered]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

I installed the ALL-IN-ONE VM IMAGE on vmware16.
when press 'ncadmin' as username and 'nextcloud' as password,it show me 'Login incorrect'
![image](https://user-images.githubusercontent.com/106236447/203705161-24ece5ab-148a-422a-8723-f101fa8eb512.png)

### Steps to reproduce

1.import nextcloud-aio.ova
2.change network conf 'nat' to 'bridge'
3.start up
4.press 'ncadmin' as username and 'nextcloud' as password

### Expected behavior

-

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### List of activated Apps

```shell
-
```

**Comment by solracsf** [comment]
Where did you download the OVA from?

**Reply by HelloMrK** [reply]
owe.
https://download.nextcloud.com/aio-vm/

**Comment by szaimen** [comment, marked as answer]
It is known that the VM image only works reliably on virtualbox. So if it is a different hypervisor, it is possible that the image will not work there. In that case you need to create your own image e.g. from a ubuntu image as base and then install AIO there.
